roles/neutron-common/templates/usr/local/bin/neutron-restart-all

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+restart neutron-server
+restart neutron-linuxbridge-agent
```

Comment:
Restarting neutron-linuxbridge-agent will restart neutron-dhcp-agent and neutron-l3-agent via service dependencies.